pkg/api/node/formatter.go

```
@@ -145,11 +151,40 @@ func (h ActionHandler) do(rw http.ResponseWriter, req *http.Request) error {
 			return apierror.NewAPIError(validation.InvalidBodyContent, fmt.Sprintf("Failed to decode request body: %v ", err))
 		}
 		return h.powerAction(toUpdate, input.Operation)
+	case enableCPUManager:
+		return h.enableCPUManager(toUpdate)
+	case disableCPUManager:
+		return h.disableCPUManager(toUpdate)
 	default:
 		return apierror.NewAPIError(validation.InvalidAction, "Unsupported action")
 	}
 }
 
+func (h ActionHandler) enableCPUManager(node *corev1.Node) error {
+	return h.requestCPUManager(node, ctlnode.CPUManagerStaticPolicy)
+}
+
+func (h ActionHandler) disableCPUManager(node *corev1.Node) error {
+	return h.requestCPUManager(node, ctlnode.CPUManagerNonePolicy)
+}
+
+func (h ActionHandler) requestCPUManager(node *corev1.Node, policy ctlnode.CPUManagerPolicy) error {
+	newNode := node.DeepCopy()
+	if newNode.Annotations == nil {
+		newNode.Annotations = make(map[string]string)
+	}
+
+	updateStatus := &ctlnode.CPUManagerUpdateStatus{
+		Status: ctlnode.CPUManagerRequestedStatus,
+		Policy: policy,
+	}
+
+	jsonStr, _ := json.Marshal(updateStatus)
```

It looks like we silent error here. It's better to return it.